geniusyield-orderbot/src/OrderBotConfig.hs

```diff
@@ -71,10 +65,6 @@ data OrderBotConfig =
     {- ^ The duration (microseconds) of time we wait before re-initiating a
          complete iteration for the bot.
     -}
-    , botCFPNftPolicy           :: FilePath
```

Comment:
All the boilerplate information is already encoded in `geniusyield-dex-api` cabal project and so is no longer required.
This change would still be backwards compatible wrt configuration as extra json fields or environment variables would simply be ignored.